config.py

```diff
@@ -1,7 +1,13 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+# Deprecate COBOT_ROOT
+if 'COBOT_ROOT' in os.environ and 'BOT_ROOT' not in os.environ:
+    logging.warning(
+        'Deprecation warning: environment variable {} is replaced by {}.'
+        .format('COBOT_ROOT', 'BOT_ROOT'))
+BOT_ROOT = os.environ.get('BOT_ROOT', os.getcwd())
```

Comment:
While we're on it, you maybe want to add another variable `BOT_ROOT_KEY = 'BOT_ROOT'`, then you don't have to retype the string in all used locations :)